### PR TITLE
Update czech.xml to v7.9.3

### PR DESCRIPTION
**Pull request by xomx**

### CODE REVIEW

File: PowerEditor/installer/nativeLang/czech.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-	- last change:  Notepad++ 7.9.1  24/Oct/2020  by Ondřej Müller (mullero@email.cz)
+	- last change:  Notepad++ 7.9.3  14/Nov/2021  by Ondřej Müller (mullero@email.cz)
```
Comment (chcg) [inline PowerEditor/installer/nativeLang/czech.xml:3]: Typo Nov instead of Jan

**Comment by donho** [comment]
Please check [v7.9.3 translation diff comparing with v7.9.2](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9455). If nothing is missed, please confirm it with me, so I will include this PR in the next release. Thank you!

**Comment by xomx** [comment]
@donho It is done.

**Comment by donho** [comment]
Thank you @xomx !

**Pull request closed without merging**